sdk/go/common/resource/plugin/plugin.go

```diff
@@ -183,6 +184,15 @@ func newPlugin(ctx *Context, pwd, bin, prefix string, kind workspace.PluginKind,
 		logging.V(9).Infof("Launching plugin '%v' from '%v' with args: %v", prefix, bin, argstr)
 	}
 
+	// Create a root span for the operation
+	opts := []opentracing.StartSpanOption{}
+	if ctx != nil && ctx.tracingSpan != nil {
+		opts = append(opts, opentracing.ChildOf(ctx.tracingSpan.Context()))
+	}
+	opts = append(opts, opentracing.Tag{Key: "pulumi-decorator", Value: prefix})
```

Comment:
It would help a lot to annotate this span with which plugin is being launched, information from `pwd, bin, prefix string, kind workspace.PluginKind, args, env`. There's slight concern env might have secrets that are not sanitized but perhaps just consider adding `kind` and `bin`

Reply:
prefix seems to already include kind eg. see this log line captured immediately above:
```
Launching plugin 'aws (resource)' from '/Users/mjeffryes/workspace/pulumi/aws/bin/pulumi-resource-aws' with args: 127.0.0.1:50937
```
I can add `bin` though